src/sentry/static/sentry/app/views/releases/utils/releaseHealthRequest.tsx

```diff
@@ -282,56 +340,75 @@ class ReleaseHealthRequest extends React.Component<Props, State> {
       ?.reduce((acc, group) => acc + group.totals[field], 0);
   };
 
-  getTimeSeries = (
-    version: string,
-    project: number,
-    display: DisplayOption,
-    healthStatsPeriod?: string
-  ) => {
-    if (healthStatsPeriod === 'auto') {
+  getTimeSeries = (version: string, project: number, display: DisplayOption) => {
+    const {healthStatsPeriod} = this.props;
+    if (healthStatsPeriod === HealthStatsPeriodOption.AUTO) {
       return this.getPeriodTimeSeries(version, project, display);
     }
 
     return this.get24hTimeSeries(version, project, display);
   };
 
   get24hTimeSeries = (version: string, project: number, display: DisplayOption) => {
-    const {totalCountByReleaseIn24h} = this.state;
+    const {totalCountByReleaseIn24h, totalCountByProjectIn24h} = this.state;
     const field = this.displayToField(display);
 
+    const intervals = totalCountByProjectIn24h?.intervals ?? [];
+
+    const projectData = totalCountByProjectIn24h?.groups.find(
+      ({by}) => by.project === project
+    )?.series[field];
+
+    const releaseData = totalCountByReleaseIn24h?.groups.find(
+      ({by}) => by.project === project && by.release === version
+    )?.series[field];
+
     return [
+      {
+        seriesName: t('Total Project'),
```

Comment:
According to Dora's PR https://github.com/getsentry/sentry/pull/24581/files Total should the last in this list 😉

Reply:
I made it this way to remain consistent with the histogram on the Issues page. We'll sync with Dora and I will consolidate it in all places depending on how we decide 👍